resources-service/create-db.js

```diff
@@ -41,7 +41,7 @@ async function create() {
         console.log(`Creating user '${resourceUsername}' to manage resources schema`);
         await createUserConnection.none(`
           CREATE USER ${resourceUsername} WITH PASSWORD '${resourcePassword}' VALID UNTIL 'infinity';
-          GRANT CONNECT, CREATE ON DATABASE hrmdb TO resources;
+          GRANT CONNECT, CREATE ON DATABASE hrmdb TO ${resourceUsername};
```

Comment:
Honestly I think we should just hardcode the resources user name like we do with HRM - I can't think of a reason why it would need to be different from DB to DB

But this is better than before, at least it's consistent

Reply:
I agree that it doesn't really matter for us... but if we were to one day expect someone else to be able to run hrm for themselves, giving them the ability to specify their own credentials wouldn't be a bad thing.